coalib/coala_main.py

```diff
@@ -20,6 +20,21 @@ def do_nothing(*args):
     return True
 
 
+ALL_ACTIONS_POSSIBLE = ['Do (N)othing', '(A)pply patch', 'Add (I)gnore comment',
```

Comment:
I assume this means it's impossible or impractical to collect action names with the current library :/